BUILDING.md

```diff
@@ -19,7 +19,7 @@ sudo gem update
 
 3) Some dependencies are added via carthage. Run:
 ```
-carthage update
+carthage checkout
```

Comment:
Okay, it took two tries to get this right.

`carthage checkout` is analogous to `pod install`.
`carthage update;carthage build` is analogous to `pod update`.